log/example_new_json_logger_test.go

```diff
@@ -0,0 +1,20 @@
+package log_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+)
+
+// JSONLogger Example
+func ExampleNewJSONLogger() {
+	var buf bytes.Buffer
+
+	logger := log.NewJSONLogger(&buf)
+	logger.Log("question", "what is the meaning of life?", "answer", 42)
+
+	fmt.Print(&buf)
+	// Output:
+	// {"answer":42,"question":"what is the meaning of life?"}
+}
```

Comment:
We should eliminate the needless buf, and (since JSON is unstable order) probably reduce the logged keyvals to 1.

```go
func ExampleJSON() {
    log.NewJSONLogger(os.Stderr).Log("meaning_of_life", 42)
    // Output:
    // {"meaning_of_life":42}
}
```